doc/userguide/src/utils/tag_stat_args.txt

```diff
@@ -1,7 +1,7 @@
 --tagstatlink owner-*:mailto:%1@domain.com?subject=Acceptance_Tests:Send_mail 
 --tagstatlink mytag:http://google.com:Google i
---tagstatlink jython-bug-*:http://jython.org/bugs/%1:Jython-bugs 
---tagstatcombine owner-* 
+--tagstatlink jython-bug-*:http://jython.org/bugs/%1:Jython-bugs
```

Comment:
I guess `jython-bug` would be fine, but changing it e.g. to `example-bug` and the domain to `example.com` would probably be a good idea anyway.